subprojects/gradle-guides-plugin/src/functionalTest/groovy/org/gradle/docs/guides/BasicGuidesDocumentationFunctionalTest.groovy

```diff
@@ -342,6 +343,7 @@ class BasicGuidesDocumentationFunctionalTest extends AbstractGuideFunctionalSpec
         indexFile.text.contains('Task :helloWorld')
     }
 
+    @Ignore("Unsupported loading resources from the plugins classpath with Asciidoctor Gradle Plugin 3.x")
```

Comment:
@jjohannes iiuc this is only used for guides and we don't use that anymore.

For the `gradle/gradle` documentation we instead define head/header/footer from resources located in the `gradle/gradle` build. I validate that this works.

Could you confirm we don't use the head/header/footer resources present in this build `src/main/resources`?

Reply:
Yes we do not render any pages anymore for the guides. We redirect to the user manual for everything now. Feel free to remove this test and maybe also "src/main/resources". 

In total this means that we no longer need the "GuidesDocumentationPlugin" part of the "BuildDocumentationPlugin". So there is quite some code we could remove I think. But that would go beyond the current effort I think.

Reply:
Thanks. I removed the minimum in this PR